google/cloud/spanner/read_partition.cc

```diff
@@ -36,6 +36,45 @@ ReadPartition::ReadPartition(std::string transaction_id, std::string session_id,
   *proto_.mutable_key_set() = spanner_internal::ToProto(std::move(key_set));
   proto_.set_limit(read_options.limit);
   proto_.set_partition_token(std::move(partition_token));
+  if (read_options.request_priority) {
+    auto* request_options = proto_.mutable_request_options();
+    switch (*read_options.request_priority) {
+      case spanner::RequestPriority::kLow:
```

Comment:
maybe too late for this, but introducing a new enum like this is less future proof than just using an alias for the enum [*].  If they ever add a `PRIORITY_URGENT` level you will need to change this code to make it usable.



[*]: Frankly I think using enums in proto APIs is not the best idea, strings or integers are easier to evolve.

Reply:
Ack.